--- Eterm-0.9.2/themes/Escreen/theme.cfg.in.fix	2004-09-22 11:12:45.122790152 +0200
+++ Eterm-0.9.2/themes/Escreen/theme.cfg.in	2004-09-22 11:13:36.268014896 +0200
@@ -76,6 +76,8 @@
 end menu
 
 begin actions
+    bind shift Left to script "es_display(prev)"
+    bind shift Right to script "es_display(next)"
     bind alt Left to script "es_display(prev)"
     bind alt Right to script "es_display(next)"
     bind alt Up to script "es_region(prev)"
@@ -124,6 +126,13 @@
     bind alt shift R to script "es_reset()"
 end actions
 
+begin keyboard
+	# Home - ^A
+	keysym 0xFF50 "\^[[1~"
+	# End - ^E
+	keysym 0xFF57 "\^[[4~"
+end keyboard
+
 begin escreen
     url ""
     #delay 2
