diff -ur Eterm-0.8.9.orig/src/command.c Eterm-0.8.9/src/command.c
--- Eterm-0.8.9.orig/src/command.c	Fri Apr 23 22:56:49 1999
+++ Eterm-0.8.9/src/command.c	Wed Jun 16 16:05:56 1999
@@ -520,8 +520,13 @@
 
 /* extern functions referenced */
 #ifdef UTMP_SUPPORT
+/*
 extern void cleanutent(void);
 extern void makeutent(const char *, const char *);
+*/
+#include <utempter.h>
+#  define cleanutent() removeFromUtmp()
+#  define makeutent(pty, hostname) addToUtmp(pty, display_name, ptyfd);
 
 #else
 #  define cleanutent() ((void)(0))
