diff -Nru Eterm-0.8.10/doc/Eterm.1.in Eterm-0.8.10.new/doc/Eterm.1.in
--- Eterm-0.8.10/doc/Eterm.1.in	Mon Nov  8 23:17:08 1999
+++ Eterm-0.8.10.new/doc/Eterm.1.in	Mon Nov  8 23:16:30 1999
@@ -1151,17 +1151,6 @@
 will work too, though it may produce an error message.
 .RE
 
-.BI term_name " name"
-.RS 5
-Use
-.I name
-as the
-.B $TERM
-environment variable, which controls which termcap/terminfo entry gets used.
-The default is
-.BR xterm .
-.RE
-
 .BI exec " command"
 .RS 5
 Rather than executing a shell, this will cause Eterm to spawn
diff -Nru Eterm-0.8.10/src/feature.h.in Eterm-0.8.10.new/src/feature.h.in
--- Eterm-0.8.10/src/feature.h.in	Mon Nov  8 23:17:08 1999
+++ Eterm-0.8.10.new/src/feature.h.in	Mon Nov  8 23:16:30 1999
@@ -418,7 +418,7 @@
 #ifdef KANJI
 # undef GREEK_SUPPORT
 # undef XTERM_FONT_CHANGE
-# undef DEFINE_XTERM_COLOR
+# undef DEFINE_XTERMi_COLOR
 # define KFONT0 "k14"
 # define KFONT1 "jiskan16"
 # define KFONT2 "jiskan18"
@@ -470,14 +470,8 @@
 
 #define APL_NAME	"Eterm"	/* The name of our beloved program */
 
-/* COLORTERM, TERM environment variables */
-#ifdef KANJI
-# define TERMENV	"kterm"
-# define COLORTERMENV	"Kterm"
-#else
-# define TERMENV	"xterm"
-# define COLORTERMENV	"Eterm"
-#endif
+/* TERM environment variables */
+#define TERMENV	"xterm-color"
 
 #ifdef NO_MOUSE_REPORT
 # ifndef NO_MOUSE_REPORT_SCROLLBAR
diff -Nru Eterm-0.8.10/src/main.c Eterm-0.8.10.new/src/main.c
--- Eterm-0.8.10/src/main.c	Mon Nov  8 23:17:08 1999
+++ Eterm-0.8.10.new/src/main.c	Mon Nov  8 23:19:26 1999
@@ -413,37 +413,13 @@
   /* add entries to the environment:
    * DISPLAY:       X display name
    * WINDOWID:      X windowid of the window
-   * COLORTERM:     Terminal supports color
-   * COLORTERM_BCE: Terminal supports BCE
    * TERM:          Terminal type for termcap/terminfo
    */
   putenv(display_string);
   putenv(windowid_string);
   if (Xdepth <= 2) {
-    putenv("COLORTERM=" COLORTERMENV "-mono");
-    putenv("COLORTERM_BCE=" COLORTERMENV "-mono");
     putenv("TERM=" TERMENV);
-  } else {
-    if (rs_term_name != NULL) {
-      i = strlen(rs_term_name);
-      term_string = MALLOC((i + 6) * sizeof(char));
-
-      sprintf(term_string, "TERM=%s", rs_term_name);
-      putenv(term_string);
-    } else {
-#ifdef DEFINE_XTERM_COLOR
-      if (Xdepth <= 2)
-	putenv("TERM=" TERMENV);
-      else
-	putenv("TERM=" TERMENV "-color");
-#else
-      putenv("TERM=" TERMENV);
-#endif
-    }
-    putenv("COLORTERM=" COLORTERMENV);
-    putenv("COLORTERM_BCE=" COLORTERMENV);
   }
-  putenv("ETERM_VERSION=" VERSION);
 
   D_CMD(("init_command()\n"));
   init_command(rs_execArgs);
diff -Nru Eterm-0.8.10/src/options.c Eterm-0.8.10.new/src/options.c
--- Eterm-0.8.10/src/options.c	Mon Nov  8 23:17:08 1999
+++ Eterm-0.8.10.new/src/options.c	Mon Nov  8 23:16:30 1999
@@ -155,7 +155,6 @@
 
 char *rs_scrollbar_type = NULL;
 unsigned long rs_scrollbar_width = 0;
-const char *rs_term_name = NULL;
 
 #if MENUBAR_MAX
 const char *rs_menubar_move = NULL;
@@ -390,7 +389,6 @@
 #if MENUBAR_MAX
       OPT_STR('M', "menu", "Default menubar file", &rs_menu),
 #endif
-      OPT_LONG("term-name", "value to use for setting $TERM", &rs_term_name),
       OPT_BOOL('C', "console", "grab console messages", NULL, &Options, Opt_console),
       OPT_ARGS('e', "exec", "execute a command rather than a shell", &rs_execArgs)
 };
@@ -828,11 +826,6 @@
 #else
   printf(" -XTERM_COLOR_CHANGE");
 #endif
-#ifdef DEFINE_XTERM_COLOR
-  printf(" +DEFINE_XTERM_COLOR");
-#else
-  printf(" -DEFINE_XTERM_COLOR");
-#endif
 #ifdef NO_MAPALERT
   printf(" +NO_MAPALERT");
 #else
@@ -2550,9 +2543,6 @@
     print_warning("Support for menus was not compiled in, ignoring");
 #endif
 
-  } else if (!BEG_STRCASECMP(buff, "term_name ")) {
-    rs_term_name = Word(2, buff);
-
   } else if (!BEG_STRCASECMP(buff, "debug ")) {
     debug_level = (unsigned int) strtoul(PWord(2, buff), (char **) NULL, 0);
 
@@ -3232,7 +3222,6 @@
   Options = (Opt_scrollBar);
   Xdisplay = NULL;
   display_name = NULL;
-  rs_term_name = NULL;
 #ifdef CUTCHAR_OPTION
   rs_cutchars = NULL;
 #endif
@@ -3628,7 +3617,6 @@
   fprintf(fp, "    min_anchor_size %d\n", rs_min_anchor_size);
   fprintf(fp, "    border_width %d\n", TermWin.internalBorder);
   fprintf(fp, "    menu %s\n", rs_menu);
-  fprintf(fp, "    term_name %s\n", getenv("TERM"));
   fprintf(fp, "    debug %d\n", debug_level);
   if (Options & Opt_exec && rs_execArgs) {
     fprintf(fp, "    exec ");
diff -Nru Eterm-0.8.10/src/options.h Eterm-0.8.10.new/src/options.h
--- Eterm-0.8.10/src/options.h	Mon Nov  8 23:17:08 1999
+++ Eterm-0.8.10.new/src/options.h	Mon Nov  8 23:16:30 1999
@@ -46,7 +46,6 @@
 extern const char  *rs_scrollBar_floating;
 extern const char  *rs_scrollbar_popup;
 extern       char  *rs_viewport_mode;
-extern const char  *rs_term_name;
 extern const char  *rs_menubar;
 extern const char  *rs_menu;
 extern const char  *rs_menubar_move;
diff -Nru Eterm-0.8.10/themes/Eterm/MAIN.in Eterm-0.8.10.new/themes/Eterm/MAIN.in
--- Eterm-0.8.10/themes/Eterm/MAIN.in	Mon Nov  8 23:17:08 1999
+++ Eterm-0.8.10.new/themes/Eterm/MAIN.in	Mon Nov  8 23:16:30 1999
@@ -192,9 +192,6 @@
 # File to read for menubar
     menu Eterm.menu
 
-# Value to use for $TERM
-    term_name xterm
-
 # Program to exec (intended for use with themes)
 #    exec foo
 
diff -Nru Eterm-0.8.10/themes/chooser/MAIN.in Eterm-0.8.10.new/themes/chooser/MAIN.in
--- Eterm-0.8.10/themes/chooser/MAIN.in	Mon Nov  8 23:17:09 1999
+++ Eterm-0.8.10.new/themes/chooser/MAIN.in	Mon Nov  8 23:16:30 1999
@@ -192,9 +192,6 @@
 # File to read for menubar
     menu chooser.menu
 
-# Value to use for $TERM
-    term_name xterm
-
 # Program to exec (intended for use with themes)
 #    exec foo
 
diff -Nru Eterm-0.8.10/themes/emacs/MAIN.in Eterm-0.8.10.new/themes/emacs/MAIN.in
--- Eterm-0.8.10/themes/emacs/MAIN.in	Mon Nov  8 23:17:09 1999
+++ Eterm-0.8.10.new/themes/emacs/MAIN.in	Mon Nov  8 23:16:30 1999
@@ -192,9 +192,6 @@
 # File to read for menubar
     menu emacs.menu
 
-# Value to use for $TERM
-    term_name xterm
-
 # Program to exec (intended for use with themes)
     exec emacs -nw
 
diff -Nru Eterm-0.8.10/themes/irc/MAIN.in Eterm-0.8.10.new/themes/irc/MAIN.in
--- Eterm-0.8.10/themes/irc/MAIN.in	Mon Nov  8 23:17:09 1999
+++ Eterm-0.8.10.new/themes/irc/MAIN.in	Mon Nov  8 23:16:30 1999
@@ -192,9 +192,6 @@
 # File to read for menubar
     menu irc.menu
 
-# Value to use for $TERM
-    term_name xterm
-
 # Program to exec (intended for use with themes)
     exec irc
 
diff -Nru Eterm-0.8.10/themes/mutt/MAIN.in Eterm-0.8.10.new/themes/mutt/MAIN.in
--- Eterm-0.8.10/themes/mutt/MAIN.in	Mon Nov  8 23:17:09 1999
+++ Eterm-0.8.10.new/themes/mutt/MAIN.in	Mon Nov  8 23:16:30 1999
@@ -192,9 +192,6 @@
 # File to read for menubar
     menu mutt.menu
 
-# Value to use for $TERM
-    term_name xterm
-
 # Program to exec (intended for use with themes)
     exec mutt
 
diff -Nru Eterm-0.8.10/themes/tn3270/MAIN.in Eterm-0.8.10.new/themes/tn3270/MAIN.in
--- Eterm-0.8.10/themes/tn3270/MAIN.in	Mon Nov  8 23:17:09 1999
+++ Eterm-0.8.10.new/themes/tn3270/MAIN.in	Mon Nov  8 23:16:30 1999
@@ -225,9 +225,6 @@
 # File to read for menubar
 #    menu tn3270.menu
 
-# Value to use for $TERM
-    term_name xterm
-
 # Program to exec (intended for use with themes)
     exec tn3270
 
diff -Nru Eterm-0.8.10/themes/trans/MAIN.in Eterm-0.8.10.new/themes/trans/MAIN.in
--- Eterm-0.8.10/themes/trans/MAIN.in	Mon Nov  8 23:17:09 1999
+++ Eterm-0.8.10.new/themes/trans/MAIN.in	Mon Nov  8 23:16:30 1999
@@ -208,9 +208,6 @@
 # File to read for menubar
     menu trans.menu
 
-# Value to use for $TERM
-    term_name xterm
-
 # Program to exec (intended for use with themes)
 #    exec foo
 
